src/screen_fader.h

```diff
@@ -0,0 +1,117 @@
+/*
+ * TheXTech - A platform game engine ported from old source code for VB6
+ *
+ * Copyright (c) 2009-2011 Andrew Spinks, original VB6 code
+ * Copyright (c) 2020-2021 Vitaly Novichkov <admin@wohlnet.ru>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef SCREEN_FADER_H
+#define SCREEN_FADER_H
+
+#include "frm_main.h"
+#include "fader.h"
+#include "global_constants.h"
+
+extern FrmMain frmMain;
+
+
+struct ScreenFader
```

Comment:
Clean 👌🏼 a good example of a simple add-on component to the base engine

Reply:
Thanks :fox_face: